lib/experimental/Lists/PersonListItem/index.tsx

```diff
@@ -0,0 +1,122 @@
+import { Button } from "@/components/Actions/Button"
+import { Icon, IconType } from "@/components/Utilities/Icon"
+import { BadgeProps } from "@/experimental/exports"
+import { PersonAvatar } from "@/experimental/Information/Avatars/PersonAvatar"
+import { DotTag, DotTagProps } from "@/experimental/Information/Tags/DotTag"
+import { RawTag, RawTagProps } from "@/experimental/Information/Tags/RawTag"
+import { InfoCircle } from "@/icons/app"
+import React from "react"
+
+export type PersonListItemProps = {
```

Comment:
can we simplify the type? I am lost with logical AND and OR props, it's quite puzzling